src/index.ts

```diff
@@ -19,6 +24,13 @@ export default class BlueButton {
   version: string;
   baseUrl: string;
 
+  // AUTH props
+  usePKCE: boolean;
+  callBackUrl: string;
+  state: string;
```

Comment:
The state, codeChallenge definitely CANNOT be properties on this top level class. A single instance of this class could be used to authorize any number of beneficiaries, whereas the state/codeChallenge are unique per bene, per auth attempt. So those will be something the 3rd party app code will have to manage/provide to our functions.

Reply:
@nbragdon Per slack question, I see this :-) Wasn't thinking about the next auth parts yet and usage of those. Will update.